src/detection/auto_calibration.py

```
@@ -0,0 +1,335 @@
+
+# # --- 호모그래피 계산 ---
+# eye_points_np = np.array(eye_points, dtype=np.float32)
+# scene_points_np = np.array(scene_points, dtype=np.float32)
+# H, status = cv2.findHomography(eye_points_np, scene_points_np)
+
+# print("\n✅ 호모그래피 행렬")
+# print(H)
+import numpy as np
+import cv2
+import os
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)